Content.Server/PDA/PdaAccessSet.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Content.Server.PDA
+{
+    public sealed class PDAAccessSet : ISet<string>
+    {
+        private readonly PDAComponent _pdaComponent;
+        private static readonly HashSet<string> EmptySet = new();
+
+        public PDAAccessSet(PDAComponent pdaComponent)
+        {
```

Comment:
Class seems unnecessary + duplicates access with AccessReaderComponent.

Reply:
It's not my code, I would leave it as is. Beside, it probably should be refactored by Access ECS

Reply:
If you're doing access that's okay, if someone else is likely to do it leave a comment for them.